ddR/R/ddR.R

```diff
@@ -246,6 +255,8 @@ dmapply <- function(FUN ,..., MoreArgs=list(),
   # Ensure that ... arguments are of equal length. length() works correctly for data.frame,
   # arrays, and lists
   lens <- vapply(dargs,function(x) {
+    # TODO Clark: Why even allow the possibility of this happening where
```

Comment:
ddR uses a global backend, so I'm not seeing how it would work to be using and switching between multiple backends in the same session.

The solution I had in mind is to allow only one backend per session (implemented inside `useBackend`), but maybe this is too restrictive?